src/components/Jobs/JobDetail.tsx

```diff
@@ -1,4 +1,73 @@
-function JobDetail() {
-  return <div className="flex h-full flex-col" />;
+import { Button, Container, Loader, Tabs } from '@mantine/core';
+import { useParams } from 'react-router-dom';
+
+import JobDescriptionTab from '@/components/Jobs/JobDescriptionTab';
+import JobOverviewTab from '@/components/Jobs/JobOverviewTab';
+import { useAuth } from '@/hooks/useAuth';
+import { useJob, useKillJobMutation } from '@/hooks/useJobs';
+
+export default function JobDetails() {
+  const { jobId = '', region = '', env = '' } = useParams();
+  const { data: jobData } = useJob(env, region, jobId);
+  const { mutate } = useKillJobMutation();
+  const { user } = useAuth();
+
+  const tabs: string[] = ['Overview', 'Job Description', 'Stage Details', 'Job Status Output'];
+  const items = tabs.map((tab) => (
+    <Tabs.Tab className={`rounded-lg bg-transparent text-gray-500`} value={tab} key={tab}>
+      {tab}
+    </Tabs.Tab>
+  ));
+
+  return (
+    <div className={'w-full'}>
+      {jobData === undefined && (
+        <div className={'flex h-screen w-full flex-row items-center justify-center'}>
+          <Loader />
+        </div>
+      )}
+
+      <div className={'absolute right-0 top-0 my-12 ml-auto mr-10 flex flex-row'}>
```

Comment:
Any reason why this needs to be absolutely positioned? I can also take a look over the weekend once I get time.

Reply:
I thought it would be easy as the overview details are in the grid adding a "state tag" in the div would disturb other elements.

Reply:
I think we should use a React portal to allow elements at the top row where the breadcrumbs are but we can fix that later in another commit.